caffe2/contrib/aten/CMakeLists.txt

```diff
@@ -18,8 +18,9 @@ if(USE_ATEN)
   ${CMAKE_CURRENT_SOURCE_DIR}/gen_op.py
   ${CMAKE_CURRENT_SOURCE_DIR}/aten_op_template.h)
 
-  add_custom_target(aten_build
-  DEPENDS aten_op.h)
+  add_custom_target(__aten_op_header_gen DEPENDS aten_op.h)
```

Comment:
IMO, this should be inside ATen CMakeLists

Reply:
This is actually generated by a separate build script (gen_op.py). ATen itself is a bit entangled at the moment... cc @zdevito @dzhulgakov